CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ More details about these changes can be found on our GitHub repo.
 ### Added
 
 * Turn off session tickets for nginx plugin by default
+* Turn off session tickets for apache plugin by default
```

Comment:
This should be moved under upcoming version (sorry for taking so long to review).

Reply:
Done.